pkg/cloudmap/instances_reconciler.go

```diff
@@ -151,6 +155,9 @@ func (r *defaultInstancesReconciler) buildInstanceAttributes(ms *appmesh.Mesh, v
 	}
 	podsNodeName := pod.Spec.NodeName
 	attr[AttrAWSInstanceIPV4] = pod.Status.PodIP
+
+	//In future, in case of multiple port mappings, the first mapping will be selected.
```

Comment:
Might be better to reword the comment as - `VirtualNode currently supports only one listener but even if support for multiple listeners in introduced, we will always derive the port value from the first listener config`.

Reply:
Implemented re-wording changes for comment.